x/bounty/client/cli/tx.go

```diff
@@ -53,9 +53,7 @@ func NewCreateProgramCmd() *cobra.Command {
 					return fmt.Errorf("internal error, failed to generate key")
 				}
 				encKey = crypto.FromECDSAPub(&decKey.ExportECDSA().PublicKey)
-
-				// TODO: avoid overwriting silently
-				SaveKey(decKey, clientCtx.HomeDir)
+				SaveKey(decKey, clientCtx.HomeDir, creatorAddr.String())
```

Comment:
I think the same creator can create multiple programs. So the saved key file also will be overwritten. How about having the program name as a filename suffix, or UUID, or something else?

Reply:
program id should be unique

Reply:
Use program id could have this issue. Running cmd of create program may failed, the program id would not increase, then the id may be used by other user.

Reply:
done by adding next program id in dec key file name.